tests/integration/cattletest/core/test_port.py

```diff
@@ -108,22 +108,20 @@ def test_container_port_start(client, sim_context, network):
     assert count == 3
 
 
-def test_container_port_stop(admin_client, sim_context, network):
+def test_container_port_stop(internal_test_client, sim_context, network):
```

Comment:
For this whole modules, it seems like we should use admin_client as much as possible for port tests, as admins can do the vast majority of the port bahavior, right?